frontend/src/views/routes/index.js

```diff
@@ -1,13 +1,15 @@
 import React from "react";
 import { Redirect, Route, Switch } from "react-router-dom";
-import Dashboard from "../dashboard";
-import ControlPanel from "../cp_control_panel/ControlPanel";
 import { AdminInstructorsView } from "../instructors";
 import { AdminSessionsView } from "../sessions";
-import { AdminPositionsView } from "../positions";
-import { AdminAssignmentsView } from "../assignments";
+import { AdminPositionsView, InstructorPositionsView } from "../positions";
+import {
+    AdminAssignmentsView,
+    InstructorAssignmentsView,
+} from "../assignments";
 import { AdminContractTemplatesView } from "../contract_template";
 import { Landing } from "../admin/landing.tsx";
```

Comment:
maybe change this to `Landing as AdminLanding` for consistency's sake?

Reply:
Yeah. That would be good. You were the one who called it `Landing` ;-)